flows/actions/call_webhook.go

```diff
@@ -11,6 +12,8 @@ import (
 	"golang.org/x/net/http/httpguts"
 )
 
+func isValidURL(u string) bool { _, err := url.Parse(u); return err == nil }
```

Comment:
see https://golang.org/cl/49930.. it's just really annoying to not be able to use `url` as a variable name